File: leetcode/day2/python/so.py
from itertools import product
from typing import List


class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        length = 0

        for word in words:
            notok = False
            for char in word:
                if (char not in allowed):
                    notok = True
            if (notok == False):
                length += 1

        return length

File: leetcode/day2/cpp/so.cpp
#include "so.h"

int Solution::countConsistentStrings(std::string allowed, std::vector<std::string> &words)
{

	unsigned int length = 0;
	for (std::string &word : words)
	{

		bool notok = false;
		for (size_t i = 0; i < word.length(); i++)
		{
			/* code */
			std::string c = word.substr(i, 1);
			if (allowed.find(c) == std::string::npos)
			{
				notok = true;
				break;
			}
		}

		if (!notok)
		{
			length++;
		}
	}

	return length;
}